### ISSUE TITLE
Toggle button should be present when editing the link

### ISSUE THREAD

**Issue by PriyansuMaurya** [urdadx/librelinks#43]
As an example, I initially added this as a Medium link, but it seems the logo doesn't display. Now, my intention is to transform it into a regular button. If there had been the same toggle button available during the initial link creation, I could have effortlessly made the change. However, I had to manually create a new one and remove the old link.

This scenario isn't unique; at times, individuals might want to convert their social media links into buttons.

![image](https://github.com/urdadx/librelinks/assets/101447544/0217cac9-4b83-46a4-85fe-ba5a41dec516)

![image](https://github.com/urdadx/librelinks/assets/101447544/7fae5d3a-371a-45e9-b68c-a83519c45d73)

![image](https://github.com/urdadx/librelinks/assets/101447544/0431d847-5e20-45dd-82fc-551eceb65189)

**Comment by urdadx** [comment]
Yeah, I think this is a great idea. In the meantime, I'm improving the algorithm that extracts the site name from the provided URL.
